library/2.1.5/validations.py

```diff
@@ -135,6 +135,13 @@ def valid_fs_path_or_raise(path: str):
     return path
 
 
+def allowed_fs_host_path_or_raise(path: str):
+    disallowed_paths = ["/mnt"]
+    if path in disallowed_paths:
```

Comment:
Gotta be a bit more strict here. We should also prevent straight up `/`.

`/mnt/ in ['/mnt'] == False`

Reply:
it is passed already with `rstrip("/")`. I'll add rstrip here as well as failsafe